server/package.json

```diff
@@ -57,6 +62,7 @@
     "prisma": "^2.19.0",
     "prisma-json-schema-generator": "^1.2.1",
     "recursive-readdir-synchronous": "^0.0.4",
+    "request-promise": "^4.2.6",
```

Comment:
https://github.com/request/request

> As of Feb 11th 2020, request is fully deprecated. No new changes are expected to land. In fact, none have landed for some time.

https://www.npmjs.com/package/request-promise

> request-promise has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142

deprecated です。

Reply:
代わりに got を使うようにしました。